mvp-0/README.md

```diff
@@ -1,117 +1,239 @@
-     -f thanos-storage.yaml # Optional
-   ```
-
-#### Deploy dnation-kubernetes-monitoring
-
-1. Clone and build rules/dashboards
-   ```bash
-   git clone https://github.com/dNationCloud/kubernetes-monitoring.git
-   cd kubernetes-monitoring
-   git checkout scs-kaas-mvp
-   make jsonnet-package
-   cd -
-   ```
-2. Deploy
-   ```bash
-   helm upgrade --install dnation-kubernetes-monitoring kubernetes-monitoring/chart --dependency-update \
-     --set releaseOverride=dnation-kubernetes-monitoring-stack \
-     -f values-observer-dash.yaml
-   ```
-
-#### Deploy kaas-metric-importer
-
-Deployment uses an image built from https://github.com/m3dbx/prometheus_remote_client_golang.
+This MVP-O aims to build an Observer monitoring solution intended to offer a global **metrics**
+view of the CSP infrastructure. It is the platform where CSP infrastructure **metrics**
+are fetched, processed, stored, and visualized. Note that this monitoring solution could
+be extended, and the other two observability signals (logs and traces) from the CSP
+infrastructure could also be processed here.
+
+_This MVP-0 builds a base for an Observer monitoring solution intended to become an **SCS product**
+in the future versions, once it attains sufficient stability._
+
+The current MVP-0 implements the initial iteration of the Observer monitoring solution,
+deployed in the Kubernetes cluster, subsequently referred to as the Observer cluster.
+As illustrative examples of how this monitoring solution can be utilized, the MVP-0 also
+implements three use cases:
+- Monitoring of the KaaS layer
+- Monitoring of the IaaS layer
+- Monitoring of infrastructure services (such as CSP services deployed on top of the IaaS layer)
+
+_The monitoring stack employed for observing the KaaS layer and infrastructure services
+is designed to serve as an **SCS product** in the future versions, once it attains sufficient stability.
+This is because the monitoring focuses on elements like Kubernetes clusters and infrastructure
+services, utilizing Kubernetes API and protocols such as HTTP(S), TCP, ICMP, etc.,
+which are aspects CSPs are unlikely to alter._
+
+TODO: insert a high leve; arch. diagram here
+
+# Deployment
+
+These deployment steps cover the process of deploying the Observer monitoring solution
+into the Kubernetes cluster. Additionally, they provide optional guidance for deploying
+a KaaS mock service, showcasing the Monitoring of the KaaS layer use case. Furthermore,
+there is an optional guide for deploying the blackbox exporter, enabling the monitoring
+of infrastructure endpoint availability.
+The monitoring of the IaaS layer use case is beyond the scope of these deployment steps.
+
+
+## Prerequisites
+
+- [kind](https://kind.sigs.k8s.io/)
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
+- [helm](https://helm.sh/)
+- [jsonnet](https://github.com/google/go-jsonnet)
+- [python 3.8+](https://www.python.org/) - needed for Monitoring of the KaaS layer use case
+
```

Comment:
`make` and `git` are missing here

Reply:
fixed: 79b0c3d76115c2278b04fe880530c25cb6a4d7fd